src/vcModals.cpp

```diff
@@ -5,6 +5,7 @@
 #include "vcThirdPartyLicenses.h"
 #include "vcPOI.h"
 #include "gl/vcTexture.h"
+#include "gl/vcFramebuffer.h"
```

Comment:
This should not be required here- viewing the framebuffer makes no sense; just save the image and load it back in?

That also means the array in `vcState` can be removed?